teuthology/task/clock.py

```diff
@@ -35,14 +35,18 @@ def task(ctx, config):
         rem.run(
             args = [
                 'sudo', 'service', 'ntp', 'stop', run.Raw('||'),
-                'sudo', 'systemctl', 'stop', 'ntp.service',
+                'sudo', 'systemctl', 'stop', 'ntp.service', run.Raw('||'),
```

Comment:
Don't `service nap stop` and `systemctl stop ntp.service` do the same?

Reply:
Yes, I'm inclined to drop the `service . . . stop` bits.